testdata/repros/Issue2723.t.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity >=0.8.0;
+
+import "ds-test/test.sol";
+import "../cheats/Cheats.sol";
+
+// https://github.com/foundry-rs/foundry/issues/2723
+contract Issue2629Test is DSTest {
+    Cheats constant vm = Cheats(HEVM_ADDRESS);
+
+    function testRollFork() public {
+        address coinbase = 0x0193d941b50d91BE6567c7eE1C0Fe7AF498b4137;
+
+        vm.createSelectFork("rpcAlias", 9);
+
+        assertEq(block.number, 9);
+        assertEq(coinbase.balance, 11250000000000000000);
+
+        vm.rollFork(10);
+
+//        assertEq(block.number, 10);
+//        assertEq(coinbase.balance, 16250000000000000000);
```

Comment:
These are supposed to not be commented, right?

Reply:
whoops, should be included, fixing